chia/farmer/farmer.py

```diff
@@ -496,12 +498,14 @@ async def update_pool_farmer_info() -> Tuple[Optional[GetFarmerResponse], Option
                                 farmer_info, farmer_is_known = await update_pool_farmer_info()
                                 if farmer_info is None and not farmer_is_known:
                                     self.log.error("Failed to update farmer info after POST /farmer.")
+                        if pool_config.payout_instructions.lower() != farmer_info.payout_instructions.lower():
+                            pool_config = replace(pool_config, payout_instructions=farmer_info.payout_instructions)
+                            just_updated = True
```

Comment:
If we change the config file, won't that trigger this to be just_updated = True?